client/js/components/data_page.jsx

```diff
@@ -3,6 +3,8 @@ var _ = require("lodash");
 var $ = require("jquery");
 var AppStore = require("../store/app_store");
 var Chart = require("./chart.jsx");
+var Papa = require("babyparse");
+var ss = require("simple-statistics");
```

Comment:
var name isn't very helpful. i get that `simpleStatistics` is verbose, but even `stats` would be a huge improvement.